test/e2e/v3/e2e_suite_test.go

```diff
@@ -0,0 +1,265 @@
+/*
```

Comment:
Hi @esric, 

would be possible take advantage that you are working on with and keep the suite_test.go just to trigger the tests? The suite_test should not have the tests actually. Could we do here something such as follows? 

- suite_test.go : env test setup, prequirements installs
- v2_plugin_cluster_test.go: for the v2 plugin tests
- v3_plugin_cluster_test.go: for the v3 plugin tests

Reply:
Sure I can rename test files.

Reply:
I have a todo note for a while to work on in its cleanups and try to do as we did with SDK tests. So, if you be able to do that will be great. However, I do not want blocker your pr because of the cleanup.

Reply:
Done.